cognite/neat/rules/exporter/rules2pydantic_models.py

```diff
@@ -495,21 +495,47 @@ def is_external_id_valid(external_id: str) -> bool:
 
     for edge_one_to_many in self.edges_one_to_many:
         edge_type_id = f"{self.__class__.__name__}.{edge_one_to_many}"
-
-        edges.extend(
-            EdgeApply(
-                space=data_model.space,
-                external_id=f"{self.external_id}-{end_node_id}",
-                type=(data_model.space, edge_type_id),
-                start_node=(data_model.space, self.external_id),
-                end_node=(
-                    data_model.space,
-                    end_node_id,
-                ),
+        if add_class_prefix:
+            edges_list = []
+            dm_view = data_model.views[self.__class__.__name__]
+            if dm_view.properties:
+                for end_node in self.__getattribute__(edge_one_to_many):
```

Comment:
Lets avoid the __gettribute__ dunder method and instead use the built in. 
```suggestion
                for end_node in getattr(self, edge_one_to_many):
```

Reply:
Same comment as above